examples/activemq/CommunityToolkit.Aspire.Hosting.ActiveMQ.MassTransit/HelloWorldConsumer.cs

```diff
@@ -0,0 +1,19 @@
+using MassTransit;
+
+namespace CommunityToolkit.Aspire.Hosting.ActiveMQ.MassTransit;
+
+public class HelloWorldConsumer : IConsumer<Message>
```

Comment:
Use a primary constructor and remove the field to be more consistent across the codebase.

Reply:
Resolved?